PatientRegistrationApp/src/App.vue

```diff
@@ -1,6 +1,14 @@
 <template>
   <v-app id="Patient-registration">
     <v-main>
+      <v-system-bar
```

Comment:
v-if with a boolean attached to our config (could read from our .env file for configuration by users of system)